lib/travis/build/templates/header.sh

```diff
@@ -12,6 +12,13 @@ ANSI_GREEN="\033[32;1m"
 ANSI_RESET="\033[0m"
 ANSI_CLEAR="\033[0K"
 
+: "${SHELL:=/bin/bash}"
+: "${TERM:=xterm-256color}"
```

Comment:
It is set to `xterm` on GCE. https://travis-ci.org/BanzaiMan/travis_production_test/builds/174902179#L458